tests/unit_tests/common.py

```
@@ -0,0 +1,209 @@
+
+class TestReadFile:
+    """Tests for the read_file tool."""
+
+    def test_read_file_success(self, setup_test_directory):
+        """Test successful file reading."""
+        test_dir = setup_test_directory
+        test_file = os.path.join(test_dir, "test_file.txt")
+        test_content = "This is test content for read_file"
+        
+        # Create a file using the create_file function
+        a = create_file(test_file, test_content)
```

Ruff (0.11.9)

Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)